### PR TITLE
Mathematical constants

### PR DESCRIPTION
**Pull request by jlconlin**

### CODE REVIEW

File: src/constants/math.hpp [hunk 1]
```diff
@@ -4,7 +4,10 @@ namespace njoy{
 namespace constants{
 namespace math{
 
-static constexpr double pi = 3.14159265358979323;
+static constexpr double pi{ 3.141592653589793 };
+static constexpr double e { 2.718281828459045 };
+static constexpr double root2{ 1.414213562373095 };
```
Comment (whaeck) [inline src/constants/math.hpp:9]: So why are we defining the square root of two as a constant? I can understand defining it for avoiding to take the square root each time, but in most cases we'll have to multiply it by something before taking the square root anyway: e.g. sqrt( 2 * E )

**Comment by whaeck** [comment]
As this is a new repository, maybe we should set up all of the automated testing at the same time?

File: src/constants/math.hpp (hunk 1 above)
Comment (jlconlin) [reply src/constants/math.hpp:9]: Just because it was was listed as one of the "basic mathematical constants" on Wikipedia.

**Comment by jlconlin** [comment]
Yeah, I'm working on the automated testing.

**Comment by jlconlin** [comment]
Testing is implemented

**Review by whaeck — APPROVED** [review]

**Pull request merged**